pkg/new-ui/v1beta1/frontend/src/app/constants/algorithms-settings.const.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  AlgorithmsEnum,
+  AlgorithmSettingType,
+} from '../enumerations/algorithms.enum';
+
+export interface AlgorithmSetting {
+  name: string;
+  value: any;
+  values?: any[];
+  type: AlgorithmSettingType;
+}
+
+export const GridSettings: AlgorithmSetting[] = [];
+
+export const RandomSearchSettings: AlgorithmSetting[] = [
+  {
+    name: 'random_state ',
+    value: 'None',
+    type: AlgorithmSettingType.STRING,
+  },
+];
+
+export const BaysianOptimisationSettings: AlgorithmSetting[] = [
+  {
+    name: 'base_estimator',
+    value: 'GP',
+    values: ['GP', 'RF', 'ET', 'GBRT'],
+    type: AlgorithmSettingType.STRING,
+  },
+  {
+    name: 'n_initial_points',
+    value: 10,
+    type: AlgorithmSettingType.INTEGER,
+  },
+  {
+    name: 'acq_func',
+    value: 'gp_hedge',
+    values: ['gp_hedge', 'LCB', 'EI', 'PI', 'EIps', 'PIps'],
+    type: AlgorithmSettingType.STRING,
+  },
+  {
+    name: 'acq_optimizer',
+    value: 'auto',
+    values: ['auto', 'sampling', 'lbfgs'],
+    type: AlgorithmSettingType.STRING,
+  },
+  {
+    name: 'random_state',
+    value: 'None',
+    type: AlgorithmSettingType.STRING,
+  },
+];
+
+export const HyperbandSettings: AlgorithmSetting[] = [];
+
+export const TPESettings: AlgorithmSetting[] = [];
+
+export const RandomSettings: AlgorithmSetting[] = [];
```

Comment:
For the random algoritm user can set `random_state`, ref: https://www.kubeflow.org/docs/components/katib/experiment/#random-search.

Reply:
Ahh I see what happened here. I mistakenly created both a [RandomSearchSettings](https://github.com/kubeflow/katib/pull/1427/files/7a2c96f94b07e72c23bdd95350c5cc8fa6ec5fce#diff-ee4ea2731c2a3259395141de9b8ab7f018b2ffaacb4714838094542838e4eba8R15) and the `RandomSettings` over here. The `RandomSearchSettings` is the correct one, but I accidentally used the `RamdomSettings` one https://github.com/kubeflow/katib/pull/1427/files/7a2c96f94b07e72c23bdd95350c5cc8fa6ec5fce#diff-ee4ea2731c2a3259395141de9b8ab7f018b2ffaacb4714838094542838e4eba8R201

I'll remove the `RandomSettings` and only use the `RandomSearchSettings`

Reply:
pushed a commit that removes the unused code. Also we've merged a proposed commit from @andreyvelich that uses the `RandomSearchSetting` so this should be resolved